forge/csrc/forge_passes.cpp

```diff
@@ -198,7 +198,9 @@ graphlib::Graph* run_pre_lowering_passes(
     recalculate_shapes(graph);
 
     // Fuse bias into matmuls
-    fuse_bias(graph);
+    if (env_as<bool>("FORGE_FUSE_MATMUL_BIAS")){
```

Comment:
It's okay to keep it disabled by default at the moment. However, let's create an issue with the label `tech dept` and link it here. In the long run, we would like to always fuse bias into matmul for added performance :))

Reply:
done!